timed/projects/views.py

```diff
@@ -64,7 +64,15 @@ class BillingTypeViewSet(ReadOnlyModelViewSet):
     ]
 
     def get_queryset(self):
-        return models.BillingType.objects.all()
+        """Don't show billing types to customers."""
+        user = self.request.user
+
+        queryset = models.BillingType.objects.all()
+
+        if models.CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Again - deny access by default, only allow if explicitly needed. This code only forbids to customers, but everyone else sees all the data